Source/MVC6/Boilerplate.AspNetCore.TagHelpers/Twitter/TwitterCard.cs

```diff
@@ -5,35 +5,73 @@
     using Microsoft.AspNetCore.Razor.TagHelpers;
 
     /// <summary>
-    /// With Twitter Cards, you can attach rich photos, videos and media experience to Tweets that drive traffic to your
-    /// website. Users who Tweet links to your content will have a "Card" added to the Tweet that's visible to all of
-    /// their followers. See https://dev.twitter.com/cards/overview. Sign up for Twitter Card analytics to see who is
-    /// sharing your site pages on Twitter. See https://analytics.twitter.com/about Validate your Twitter cards. See
-    /// https://twitter.com/login?redirect_after_login=https%3A%2F%2Fcards-dev.twitter.com%2Fvalidator
+    /// With Twitter Cards, you can attach rich photos, videos and media experience to Tweets that
+    /// drive traffic to your website. Users who Tweet links to your content will have a "Card" added
+    /// to the Tweet that's visible to all of their followers. See
+    /// https://dev.twitter.com/cards/overview. Sign up for Twitter Card analytics to see who is
+    /// sharing your site pages on Twitter. See https://analytics.twitter.com/about Validate your
+    /// Twitter cards. See https://twitter.com/login?redirect_after_login=https%3A%2F%2Fcards-dev.twitter.com%2Fvalidator
     /// </summary>
     public abstract class TwitterCard : TagHelper
     {
         /// <summary>
-        /// The username attribute name.
+        /// The Twitter user ID of content creator.
         /// </summary>
-        protected const string UsernameAttributeName = "username";
+        protected const string CreatorTwitterIdAttributeName = "creator-id";
 
         /// <summary>
-        /// Gets the type of the Twitter card.
+        /// The Twitter @username of content creator e.g. @RehanSaeedUK.
         /// </summary>
-        /// <value>The type.</value>
-        public abstract TwitterCardType Type { get; }
+        protected const string CreatorUsernameAttributeName = "creator";
+
+        /// <summary>
+        /// The Twitter @username associated with the page e.g. @Microsoft. This is a required
+        /// property. Required for Twitter Card analytics.
+        /// </summary>
+        protected const string TwitterSiteAttributeUsername = "site";
```

Comment:
Can we rename this to `SiteAttributeName` to be consistent with the other constants.